lua/nvim-web-devicons.lua

```diff
@@ -1373,6 +1379,7 @@ local filetypes = {
   ["dockerfile"] = "dockerfile",
   ["dosbatch"] = "bat",
   ["dosini"] = "ini",
+  ["drools"] = "drl",
```

Comment:
This should be `["drools"] = “drools"`

Reply:
I recognized I messed up testing this properly, and submitted https://github.com/kyazdani42/nvim-web-devicons/pull/137 , where I keep `["drools"] = "drl",` but then I rename `  ["drools"] = {` to `  ["drl"] = {` . I tried the way you suggested here, but that doesn't work for some reason. The new PR does work, though. There appears to be a file extension association in the icon name.

Reply:
Ah, yeah if the extension is `drl` then yes that should be used as the icon name. That could probably be clarified with a comment above the icons table.